### DISCUSSION TITLE
Look & Feel

### DISCUSSION THREAD

**Discussion by maciekpaprocki** [pocketbase/pocketbase#4931] [Q&A] [answered]
First of all. I am big fan of Pocketbase. Thank you very much for the stack, it's amazing. 

I am in the midst of writing some open-source templates for my project based on Symfony/Twig. I love pocketbase look & feel and was hopping to steal some of its components.

I know the project is under MIT. I know I can legally do it, but ethically I would prefer to have OK from your side. Can i please have one? :) 

Also, for further research are there any inspiration/open source designs or any other inspirations you used?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, feel free to take the pieces you need but keep in mind that the UI components implementation is not that great at the moment.

It will be refactored and a standalone UI components library will be created to assist with the development of Admin UI plugins (see https://github.com/pocketbase/pocketbase/discussions/4355).

**Comment by maciekpaprocki** [comment]
Thank you very much. The library will be using tailwind and twig so I need to hand-make it, but I was looking for some good look and feel and your one is the best I found. It's hard to do things without designer and it's hard to sell them on open source :)
